mappings/net/minecraft/block/MapColor.mapping

```diff
@@ -1,4 +1,5 @@
-CLASS net/minecraft/class_3620 net/minecraft/block/MaterialColor
+CLASS net/minecraft/class_3620 net/minecraft/block/MapColor
+	COMMENT The color that a block will have on a {@link net.minecraft.item.FilledMapItem map item}. Selected by the Material the block uses.
```

Comment:
```suggestion
	COMMENT Represents the surface color of a block when rendered from a {@link net.minecraft.item.map.MapState}.
```

Reply:
"Selected by the Material the block uses." is redundant and confusing. That is a property of materials, not the color. And a map color can be specified for a block that is unrelated to the material.